tool/gocross/autoflags.go

```diff
@@ -169,6 +169,7 @@ func autoflagsForTest(argv []string, env *Environment, goroot, nativeGOOS, nativ
 	env.Set("CC", "cc")
 	env.Set("TS_LINK_FAIL_REFLECT", boolStr(failReflect))
 	env.Set("GOROOT", goroot)
+	env.Set("GOTOOLCHAIN", "local")
```

Comment:
Add a comment pointing at the docs for GOTOOLCHAIN? I feel like I'll forget what this is almost immediately without a pointer.